### PR TITLE
docs: separate README file for each package

### PR DESCRIPTION
**Pull request by Salet**
https://pubnub.atlassian.net/browse/UI-1524

This introduces 3 separate README.md files: 
- one for React package, it will be displayed here: https://www.npmjs.com/package/@pubnub/react-chat-components and when browsing to `packages/react` in GitHub repo
- one for React Native package, it will be displayed here: https://www.npmjs.com/package/@pubnub/react-native-chat-components and when browsing to `packages/react-native` in GitHub repo
- a general one combining the two that is only shown in the repository root: https://github.com/pubnub/react-chat-components

This PR also introduces a LICENSE file for React Native package and a change in default formatting for markdown files - these will not be wrapping sentences anymore since it was messing up the display of the tables.

### CODE REVIEW

File: packages/react-native/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+PubNub Chat Components for React Native are the fastest way to add chat features like direct and group messaging, typing indicators, reactions, without going through the complexity of low-level architecture of realtime networks.
+
+- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using predefined components.
+- **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts, reactions etc. without writing complex code.
+- **Flexible and Customizable Components**: Customize component design and add custom components to extend functionality.
+- **High Scalability**: Let PubNub take care of scaling and reliability as you grow your app.
+- **Easy Theming**: Use the built-in light and dark themes or create custom ones for various use cases: group, support, and event chats.
+- **Strong Typing**: Utilize the power of TypeScript to develop your application.
+
+> You can also check out our [PubNub Chat Components for React](https://www.npmjs.com/package/@pubnub/react-chat-components)
```
Comment (Salet) [inline packages/react-native/README.md:12]: I wanted to have a cross-link between packages on npm so people are aware of the existence of separate packages before they land on our docs. Wondering if there's better wording for this

**Review by piotr-suwala — APPROVED** [review]

File: packages/react-native/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# PubNub Chat Components for React Native
+
+PubNub Chat Components for React Native are the fastest way to add chat features like direct and group messaging, typing indicators, reactions, without going through the complexity of low-level architecture of realtime networks.
```
Comment (kazydek) [inline packages/react-native/README.md:3]:
```suggestion
PubNub Chat Components for React Native are the fastest way to add chat features like direct and group messaging or typing indicators, without going through the complexity of the low-level architecture of real-time networks.
```

File: packages/react/README.md [hunk 3]
```diff
@@ -1,26 +1,21 @@
...
-messaging, typing indicators, reactions, without going through the complexity of low-level
-architecture of realtime networks.
-
-- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using
-  predefined components.
-- **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts,
-  reactions etc. without writing complex code.
-- **Flexible and Customizable Components**: Customize component design and add custom components to
-  extend functionality.
+PubNub Chat Components for React are the fastest way to add chat features like direct and group messaging, typing indicators, reactions, without going through the complexity of low-level architecture of realtime networks.
```
Comment (kazydek) [inline packages/react/README.md:3]:
```suggestion
PubNub Chat Components for React are the fastest way to add chat features like direct and group messaging, typing indicators, or reactions, without going through the complexity of the low-level architecture of real-time networks.
```

File: packages/react-native/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using predefined components.
+- **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts, reactions etc. without writing complex code.
```
Comment (kazydek) [inline packages/react-native/README.md:6]:
```suggestion
- **Fast and Simple Extensibility**: Add rich features, like typing indicators or read receipts, without writing complex code.
```

File: packages/react/README.md [hunk 5]
```diff
@@ -1,26 +1,21 @@
...
+
+- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using predefined components.
+- **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts, reactions etc. without writing complex code.
```
Comment (kazydek) [inline packages/react/README.md:6]:
```suggestion
- **Fast and Simple Extensibility**: Add rich features, like typing indicators, read receipts, or reactions, without writing complex code.
```

File: packages/react-native/README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+# PubNub Chat Components for React Native
+
+PubNub Chat Components for React Native are the fastest way to add chat features like direct and group messaging, typing indicators, reactions, without going through the complexity of low-level architecture of realtime networks.
+
+- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using predefined components.
```
Comment (kazydek) [inline packages/react-native/README.md:5]:
```suggestion
- **Reduced Implementation Time**: Develop proof-of-concept and production-ready apps faster using predefined components.
```

File: packages/react/README.md [hunk 7]
```diff
@@ -1,26 +1,21 @@
...
-
-- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using
-  predefined components.
-- **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts,
-  reactions etc. without writing complex code.
-- **Flexible and Customizable Components**: Customize component design and add custom components to
-  extend functionality.
+PubNub Chat Components for React are the fastest way to add chat features like direct and group messaging, typing indicators, reactions, without going through the complexity of low-level architecture of realtime networks.
+
+- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using predefined components.
```
Comment (kazydek) [inline packages/react/README.md:5]:
```suggestion
- **Reduced Implementation Time**: Develop proof-of-concept and production-ready apps faster using predefined components.
```

File: packages/react-native/README.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+> You can also check out our [PubNub Chat Components for React](https://www.npmjs.com/package/@pubnub/react-chat-components)
+
+![PubNub Chat Components](screenshot-mobile.png)
+
+## Features
+
+- **User and Channel Metadata**: Add additional information about the users, channels, and their memberships from PubNub Objects storage using custom hooks
+- **Subscriptions**: Subscribe to user channels automatically
+- **Messages**: Publish and display new and historical text messages
+- **Presence**: Get currently active users, observe their state, and notify about changes
```
Comment (kazydek) [inline packages/react-native/README.md:21]:
```suggestion
- **Presence**: Get currently active users, observe their state, and notify them about changes
```

File: packages/react/README.md [hunk 9]
```diff
@@ -1,26 +1,21 @@
...
+![PubNub Chat Components](screenshot-desktop.png)
 
 ## Features
 
-- **User and Channel Metadata**: Add additional information about the users, channels, and their
-  memberships from PubNub Objects storage using custom hooks
+- **User and Channel Metadata**: Add additional information about the users, channels, and their memberships from PubNub Objects storage using custom hooks
 - **Subscriptions**: Subscribe to user channels automatically
 - **Messages**: Publish and display new and historical text messages
 - **Presence**: Get currently active users, observe their state, and notify about changes
```
Comment (kazydek) [inline packages/react/README.md:21]:
```suggestion
- **Presence**: Get currently active users, observe their state, and notify them about changes
```

File: packages/react/README.md [hunk 10]
```diff
@@ -1,26 +1,21 @@
...
+- **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts, reactions etc. without writing complex code.
+- **Flexible and Customizable Components**: Customize component design and add custom components to extend functionality.
 - **High Scalability**: Let PubNub take care of scaling and reliability as you grow your app.
-- **Easy Theming**: Use the built-in light and dark themes or create custom ones for various use
-  cases: group, support, and event chats.
+- **Easy Theming**: Use the built-in light and dark themes or create custom ones for various use cases: group, support, and event chats.
 - **Strong Typing**: Utilize the power of TypeScript to develop your application.
 
-![PubNub Chat Components](https://i.imgur.com/992eLO8.png)
+> You can also check out our [PubNub Chat Components for React Native](https://www.npmjs.com/package/@pubnub/react-native-chat-components)
```
Comment (kazydek) [inline packages/react/README.md:12]:
```suggestion
```

File: packages/react/README.md [hunk 11]
```diff
@@ -1,26 +1,21 @@
...
+
+![PubNub Chat Components](screenshot-desktop.png)
```
Comment (kazydek) [inline packages/react/README.md:14]:
```suggestion
![PubNub Chat Components](screenshot-desktop.png)

> Apart from PubNub Chat Components for React, we provide a separate npm package for React Native. Check out our [PubNub Chat Components for React Native](https://www.npmjs.com/package/@pubnub/react-native-chat-components) for more details.
```

File: packages/react-native/README.md (hunk 1 above)
Comment (kazydek) [inline packages/react-native/README.md:12]:
```suggestion
```

File: packages/react-native/README.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using predefined components.
+- **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts, reactions etc. without writing complex code.
+- **Flexible and Customizable Components**: Customize component design and add custom components to extend functionality.
+- **High Scalability**: Let PubNub take care of scaling and reliability as you grow your app.
+- **Easy Theming**: Use the built-in light and dark themes or create custom ones for various use cases: group, support, and event chats.
+- **Strong Typing**: Utilize the power of TypeScript to develop your application.
+
+> You can also check out our [PubNub Chat Components for React](https://www.npmjs.com/package/@pubnub/react-chat-components)
+
+![PubNub Chat Components](screenshot-mobile.png)
```
Comment (kazydek) [inline packages/react-native/README.md:14]:
```suggestion
![PubNub Chat Components](screenshot-mobile.png)

> Apart from PubNub Chat Components for React Native, we provide a separate npm package for React. Check out our [PubNub Chat Components for React](https://www.npmjs.com/package/@pubnub/react-chat-components) for more details.
```

File: packages/react-native/README.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+
+## Features
+
+- **User and Channel Metadata**: Add additional information about the users, channels, and their memberships from PubNub Objects storage using custom hooks
+- **Subscriptions**: Subscribe to user channels automatically
+- **Messages**: Publish and display new and historical text messages
+- **Presence**: Get currently active users, observe their state, and notify about changes
+- **Typing Indicators**: Display notifications that users are typing
+- **Message Reactions**: Publish and add emojis to messages
```
Comment (kazydek) [inline packages/react-native/README.md:23]:
```suggestion
```

File: samples/react-native/getting-started/README.md [hunk 14]
```diff
@@ -1,35 +1,23 @@
 # Getting Started
 
-`getting-started` is a simple app that allows you to get a 1:1 chat app quickly up and running
-through the PubNub Network. It features a simple chat window with a message input field. After
-starting the app, you can send sample messages as a default `myFirstUser` by typing them in an input
-field and messages will pile up on the screen as you send them.
+`getting-started` is a simple app that allows you to get a 1:1 chat app quickly up and running through the PubNub Network. It features a simple chat window with a message input field. After starting the app, you can send sample messages as a default `myFirstUser` by typing them in an input field and messages will pile up on the screen as you send them.
```
Comment (kazydek) [inline samples/react-native/getting-started/README.md:3]:
```suggestion
`getting-started` is a simple app that allows you to get a 1:1 chat app quickly up and running through the PubNub Network. It features a simple chat window with a message input field. After starting the app, you can send sample messages as the default `myFirstUser` by typing them in an input field, and messages will pile up on the screen as you send them.
```

File: samples/react/getting-started/README.md [hunk 15]
```diff
@@ -1,34 +1,23 @@
 # Getting Started
 
-`getting-started` is a simple app that allows you to get a 1:1 chat app quickly up and running
-through the PubNub Network. It features a simple chat window with a message input field. After
-starting the app, you can send sample messages as a default `myFirstUser` by typing them in an input
-field and messages will pile up on the screen as you send them.
+`getting-started` is a simple app that allows you to get a 1:1 chat app quickly up and running through the PubNub Network. It features a simple chat window with a message input field. After starting the app, you can send sample messages as a default `myFirstUser` by typing them in an input field and messages will pile up on the screen as you send them.
```
Comment (kazydek) [inline samples/react/getting-started/README.md:3]:
```suggestion
`getting-started` is a simple app that allows you to get a 1:1 chat app quickly up and running through the PubNub Network. It features a simple chat window with a message input field. After starting the app, you can send sample messages as the default `myFirstUser` by typing them in an input field, and messages will pile up on the screen as you send them.
```

File: samples/react/telehealth/README.md [hunk 16]
```diff
@@ -41,8 +31,7 @@ Follow the steps to run the app locally.
    yarn
    ```
 
-1. Copy the `samples/.env.example` file as `samples/.env` and paste your Publish and Subscribe Keys
-   there.
+1. Copy the `samples/.env.example` file as `samples/.env` and paste your Publish and Subscribe Keys there.
 
 1. Go to the `samples/react/telehealth` folder.
```
Comment (kazydek) [inline samples/react/telehealth/README.md:37]: Could we also add similar info here? https://github.com/pubnub/chat-components-ios-examples/tree/master/telehealth-example#usernames

Comment (kazydek) [reply samples/react/telehealth/README.md:37]: Let's also link to https://www.pubnub.com/tutorials/cross-platform-chat-application-telehealth-react/

File: packages/react/README.md (hunk 3 above)
Comment (Salet) [reply packages/react/README.md:3]: fixed

File: packages/react-native/README.md (hunk 4 above)
Comment (Salet) [reply packages/react-native/README.md:6]: fixed

File: packages/react/README.md (hunk 5 above)
Comment (Salet) [reply packages/react/README.md:6]: fixed

File: packages/react-native/README.md (hunk 6 above)
Comment (Salet) [reply packages/react-native/README.md:5]: fixed

File: packages/react-native/README.md (hunk 13 above)
Comment (Salet) [reply packages/react-native/README.md:23]: fixed

File: samples/react-native/getting-started/README.md (hunk 14 above)
Comment (Salet) [reply samples/react-native/getting-started/README.md:3]: fixed

File: samples/react/getting-started/README.md (hunk 15 above)
Comment (Salet) [reply samples/react/getting-started/README.md:3]: fixed

File: samples/react/telehealth/README.md (hunk 16 above)
Comment (Salet) [reply samples/react/telehealth/README.md:39]: I added links to tutorials in a README for each of the sample apps having a tutorial, as well as to the main repo README that has the table with links to all apps

Comment (Salet) [reply samples/react/telehealth/README.md:39]: And I don't think the note about the `.env` file is applicable to the iOS repository, they probably use some other method

File: packages/react-native/README.md [hunk 17]
```diff
@@ -0,0 +1,55 @@
...
+![PubNub Chat Components](screenshot-mobile.png)
+
+> Apart from PubNub Chat Components for React Native, we provide a separate npm package for React. Check out our [PubNub Chat Components for React](https://www.npmjs.com/package/@pubnub/react-chat-components) for more details.
+
+## Features
+
+- **User and Channel Metadata**: Add additional information about the users, channels, and their memberships from PubNub Objects storage using custom hooks
+- **Subscriptions**: Subscribe to user channels automatically
+- **Messages**: Publish and display new and historical text messages
+- **Presence**: Get currently active users, observe their state, and notify about changes
```
Comment (kazydek) [inline packages/react-native/README.md:21]:
```suggestion
- **Presence**: Get currently active users, observe their state, and notify them about changes
```

File: packages/react/README.md [hunk 18]
```diff
@@ -1,26 +1,21 @@
...
+> Apart from PubNub Chat Components for React, we provide a separate npm package for React Native. Check out our [PubNub Chat Components for React Native](https://www.npmjs.com/package/@pubnub/react-native-chat-components) for more details.
 
 ## Features
 
-- **User and Channel Metadata**: Add additional information about the users, channels, and their
-  memberships from PubNub Objects storage using custom hooks
+- **User and Channel Metadata**: Add additional information about the users, channels, and their memberships from PubNub Objects storage using custom hooks
 - **Subscriptions**: Subscribe to user channels automatically
 - **Messages**: Publish and display new and historical text messages
 - **Presence**: Get currently active users, observe their state, and notify about changes
```
Comment (kazydek) [inline packages/react/README.md:21]:
```suggestion
- **Presence**: Get currently active users, observe their state, and notify them about changes
```

File: samples/react/telehealth/README.md (hunk 16 above)
Comment (kazydek) [reply samples/react/telehealth/README.md:39]: @Salet I meant this part: Screenshot 2022-10-27 at 15 27 49

Comment (Salet) [reply samples/react/telehealth/README.md:39]: This will be added to the branch containing changes to the telehealth app

**Review by kazydek — APPROVED** [review]

**Pull request merged**